src/main/java/com/oath/halodb/HaloDBOptions.java

```diff
@@ -14,6 +14,8 @@
 
     private int maxFileSize = 1024 * 1024; /* 1mb file recordSize */
 
+    private int maxTombstoneFileSize = 0; /* use maxFileSize by default unless set explicitly */
```

Comment:
The comments may confuse people, since the default value set to 0, but we actually want it to be `maxFileSize`.

Reply:
Yes, a little bit if not check ```getMaxTombstoneFileSize```. ;) I will update the comment